src/main/java/frc/robot/subsystems/EndEffector.java

```diff
@@ -5,11 +5,21 @@
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import com.revrobotics.CANSparkMax;
 
 public class EndEffector extends SubsystemBase {
+  private CANSparkMax m_RightIOMotor = new CANSparkMax(Constants.kRightIOMotorID);
+  private CANSparkMax m_LeftIOMotor = new CANSparkMax(Constants.kLeftIOMotorID);
+  private CANSparkMax m_RightHoldingMotor = new CANSparkMax(Constants.kRightHoldingMotorID);
+  private CANSparkMax m_LeftHoldingMotor = new CANSparkMax(Constants.kRightIOLeftHoldingMotorID);
```

Comment:
do these need the motor type just like the drive motor issue we fixed earlier today in drivetrain subsystem?

Reply:
They probably do

Reply:
are you saying these are the same things we deleted from drive train a couple of minutes ago? if so then no, we dont need them.

Reply:
no we fixed the arguments for the constructor so that we supplied the motor type earlier, right? I imagine we might need to include the motor type with these too, right?

Reply:
if it doesn't work one way, try the other way.

Reply:
so do i, but i think if this is the only problem we should just merge into main so we can fix it. only Julien has access  to the branch currently, so it would be much easier to fix if we merge now. again assuming this is the only problem.